src/Microsoft.Health.Fhir.Shared.Api/Modules/FeatureFlags/HtmlUi/HtmlUiFeatureModule.cs

```diff
@@ -36,6 +39,12 @@ public void Load(IServiceCollection services)
                     .Singleton()
                     .AsSelf()
                     .AsService<TextOutputFormatter>();
+
+                // Adds provider to serve embedded razor views
+                services.Configure<MvcRazorRuntimeCompilationOptions>(options =>
+                {
+                    options.FileProviders.Add(new EmbeddedFileProvider(typeof(CodePreviewModel).Assembly));
```

Comment:
I think we can add the razer view compilation here

```suggestion
                services.Configure<MvcRazorRuntimeCompilationOptions>(options =>
                {
                    options.FileProviders.Add(new EmbeddedFileProvider(typeof(CodePreviewModel).Assembly)).AddMvc().AddRazorRuntimeCompilation();
```